libraries/networking/src/AddressManager.cpp

```diff
@@ -412,13 +438,13 @@ bool isPossiblePlaceName(QString possiblePlaceName) {
 }
 
 void AddressManager::handleLookupString(const QString& lookupString, bool fromSuggestions) {
+    QString trimmedString = lookupString.trimmed();
 
-    QString sanitizedString = lookupString.trimmed();
-    if (!sanitizedString.isEmpty()) {
+    if (!trimmedString.isEmpty()) {
         resetConfirmConnectWithoutAvatarEntities();
 
         // make this a valid hifi URL and handle it off to handleUrl
-        handleUrl(sanitizedString, fromSuggestions ? Suggestions : UserInput);
+        handleUrl(trimmedString, fromSuggestions ? Suggestions : UserInput, trimmedString);
```

Comment:
It's doable to just have the first string maybe, the only issue is that I would have to modify all the other functions calling into it in order to have them continue working.

Some functions are passing in QUrls instead of QStrings.

Reply:
Modify to make the first parameter a `QString`? It shouldn't be necessary to change the first parameter from a `QUrl` to a `QString`? Something along the lines of ...
```
handleUrl(QUrl(trimmedString.replace(" ", "%20"), fromSuggestions ? Suggestions : UserInput);
```

Reply:
That doesn't appear to be having the intended effect. One thing that I could try is fill in some string that isn't %20 but valid in a domain name but is unlikely to ever be used in a Place name such as "-V-I-R-C-A-D-I-A-space-" lol and then convert it on the other end maybe and see if that works.

Reply:
OK, the current PR code is an improvement that can be merged now, I think. I'll be in this code area in the not too distant future and may take another look then.